<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gps</class>
 <widget class="QDialog" name="Gps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>224</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pbCancel">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>CANCEL</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbEnter">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>ENTER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GPS SETTING</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbMaol">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>ACTIVE</string>
   </property>
   <widget class="QRadioButton" name="rbYes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbNo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbWm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>MODE</string>
   </property>
   <widget class="QRadioButton" name="rbCm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CAR MODE</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Settings-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK____/image/wifi.png</normaloff>../Settings-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK____/image/wifi.png</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbWm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>WAITING MODE</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Settings-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK____/image/wifi.png</normaloff>../Settings-build-desktop-Desktop_Qt_4_8_0_for_GCC__Qt_SDK____/image/wifi.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
